opal/corelib/kernel.rb

```diff
@@ -1114,4 +1114,9 @@ def to_s
   end
 
   alias untaint taint
+
+  def Rational(*args)
+    #Just a stub to let unrelated rubyspecs run.
+    nil
```

Comment:
I had some stuff in my numeric branch for that, I just have to find the will to live for doing all that bridge rework actually work.